queries.sql

```diff
@@ -66,4 +66,99 @@ UPDATE animals
 SET species = 'unspecified';
 SELECT species from animals;
 ROLLBACK;
-SELECT species from animals;
\ No newline at end of file
+SELECT species from animals;
+
+ALTER TABLE animals
+DROP COLUMN species;
+
+ALTER TABLE animals
+ADD CONSTRAINT fk_owner
+FOREIGN KEY (owner_id)
+REFERENCES owners(id);
+
+ALTER TABLE animals
+ADD COLUMN owner_id integer;
+
+ALTER TABLE animals
+ADD CONSTRAINT fk_species
+FOREIGN KEY (species_id)
+REFERENCES species(id);
+
+ALTER TABLE animals
+ADD COLUMN species_id integer;
+
+CREATE TABLE species (
+    id serial PRIMARY KEY,
+    name varchar(255)
+);
+
+CREATE TABLE owners (
+    id serial PRIMARY KEY,
+    full_name varchar(255),
+    age integer
+);
```

Comment:
Everything that changes the table structure should be put in the `schema .sql` file based on the project requremnt. Please  move these code to the `schema.sql` file.
![image](https://github.com/BezzB/vetclinic/assets/101523582/60d41c5f-8f46-41ec-a38c-8a601a86050a)